packages/common/src/utils/conditionalFormatting.ts

```diff
@@ -53,32 +56,49 @@ export const createConditionalFormattingConfigWithColorRange = (
     },
 });
 
+const convertFormattedValue = (
+    value: unknown,
+    field: Field | TableCalculation | undefined,
+) => {
+    if (!field) return value;
+    if (!isField(field)) return value;
```

Comment:
Are table calculations with percent out of scope ? 
![Screenshot 2023-07-31 at 09 14 30](https://github.com/lightdash/lightdash/assets/9117144/5bba4a6e-3dea-4320-8cb7-853448a345a9)

Reply:
good catch, updated...

<img width="218" alt="CleanShot 2023-07-31 at 12 55 01@2x" src="https://github.com/lightdash/lightdash/assets/962095/21cad40c-8e67-4d4f-94fe-f02c861669bd">
<img width="474" alt="CleanShot 2023-07-31 at 12 54 58@2x" src="https://github.com/lightdash/lightdash/assets/962095/dafdd2c3-303e-4f2d-9382-7ac31e432a09">